bundles/org.openhab.voice.porcupineks/src/main/java/org/openhab/voice/porcupineks/internal/PorcupineKSConstants.java

```diff
@@ -25,7 +25,7 @@
     /**
      * Service name
      */
-    public static final String SERVICE_NAME = "Porcupine Keyword Spotter";
+    public static final String SERVICE_NAME = "Porcupine";
```

Comment:
Ok fine but in this case please add "Keyword Spotter" to label in `@ConfigurableService(category`.
See how it is done in other TTS or STT service.

Reply:
I have the feeling I have to move it to somewhere else, thank you!